R/dataframe-s3-base.R

```diff
@@ -126,26 +126,23 @@ tail.polars_data_frame <- function(x, n = 6L, ...) x$tail(n = n)
 # https://tibble.tidyverse.org/articles/invariants.html#column-subsetting
 # TODO: add document
 #' @export
-`[.polars_data_frame` <- function(x, i, j, drop = FALSE) {
+`[.polars_data_frame` <- function(x, i, j, ..., drop = FALSE) {
```

Comment:
Why do you add dots? `[.tbl_df` adds some at the end but says in the docs they are ignored, and `[.data.frame` doesn't have any. I also don't see in the test which cases are enabled by this.

Reply:
Looking at the Extract documentation, the default Extract requires that the `drop` be a named argument.
Perhaps many users are not even aware of the existence of this argument and may be confused if it is positionable.